ui/src/views/Dashboard.vue

```diff
@@ -1492,7 +1529,8 @@ export default {
         Host: this.config.leaderNode,
         User: this.config.adminUsername,
         Password: this.config.adminPassword,
-        TLSVerify: this.config.tlsVerify ? "enabled" : "disabled"
+        TLSVerify: this.config.tlsVerify ? "enabled" : "disabled",
+        LdapUserDomain: this.config.ldapUserDomain
```

Comment:
Can we provide a good default?

Reply:
no, I would prefer to stop if we do not have a good default base DN, we cannot and probably will never handle the DN rename in NS8, so let's ask to the people whet they want . just my opinion, I can change the code obviously